src/Coordination/RocksDBContainer.h

```diff
@@ -209,38 +183,58 @@ struct RocksDBContainer
         }
     }
 
-    std::vector<std::pair<std::string, Node>> getChildren(const std::string & key_, bool read_data = false)
+    std::vector<std::pair<std::string, Node>> getChildren(const std::string & key_, bool read_meta = true, bool read_data = false)
     {
         rocksdb::ReadOptions read_options;
         read_options.total_order_seek = true;
 
         std::string key = key_;
         if (!key.ends_with('/'))
             key += '/';
-        size_t len = key.size() + 2;
+        size_t len = key.size();
```

Comment:
`key_` and `key` names are extremely confusing
seems like `key` is used only for `prefix` so maybe rename it to something like `prefix_holder` or key_as_prefix`